src/v2/Apps/Artist/Routes/AuctionResults/Components/KeywordFilter.tsx

```diff
@@ -0,0 +1,36 @@
+import { LabeledInput, MagnifyingGlassIcon } from "@artsy/palette"
+import { debounce } from "lodash"
+import React, { useEffect, useMemo, useRef } from "react"
+import { useAuctionResultsFilterContext } from "../AuctionResultsFilterContext"
+
+const DEBOUNCE_DELAY = 300
```

Comment:
**Note:** The debounce delay can be adjusted, but 300 seems to make sense to me.